src/main/java/ProcessInputException.java

```diff
@@ -0,0 +1,2 @@
+public class ProcessInputException extends Exception{
+}
```

Comment:
Empty Class. Please add at least a constructor with valid instantiations/assignments, so if this error is thrown, the description shown in correct.